esrally/track/loader.py

```diff
@@ -204,6 +211,8 @@ def __init__(self, cfg, fetch, update, repo_class=repo.RallyRepository):
                 self.repo.checkout(repo_revision)
             else:
                 self.repo.update(distribution_version)
+                track_revision = git.head_revision(self.track_dir(self.track_name))
```

Comment:
Same comment here as earlier, if you move this inside `RallyRepository` you won't have to directly use `git` here, but just get the info from the instance attribute.